feg/gateway/services/aaa/test/pipelined/main.go

```diff
@@ -60,17 +60,10 @@ func (c *DummyPipelined) ActivateFlows(
 	ctx context.Context, req *protos.ActivateFlowsRequest) (res *protos.ActivateFlowsResult, err error) {
```

Comment:
can you remind me why do we have code on `ActivateFlows`? I can't see this function is used at all by any go service

Are you using this mock pipelined to test something?

Reply:
no idea why its there

Reply:
I'm guessing just for completeness

Reply:
That is fine. But if you want to delete it completely it is fine. This way you don't have to come here anymore if you ever change the rules again.  

Up to you